### DISCUSSION TITLE
Submit a new theme?

### DISCUSSION THREAD

**Discussion by Lucas3oo** [labithiotis/zed-themes#49] [Q&A] [answered]
How do I submit a new theme to https://zed-themes.com/?

**Comment by labithiotis** [comment]
@Lucas3oo Currently the themes are pulled periodically from [zed extension](https://github.com/zed-industries/extensions) repo. I'm working on ability to create/publish/manage your own themes directly in zed-themes, maybe within the week I will have something.

**Reply by labithiotis** [reply, marked as answer]
You can now publish new themes by logging in
